Thomasboevith patch 1

thomasboevith opened the pull request:
I have added support for Danish language to streams.asm and added a font file and a font license file. I changed the Swedish font file so that ä/Ä became æ/Æ and ö/Ö became ø/Ø (see the image below).

![image](https://user-images.githubusercontent.com/11942066/66309159-308c7680-e909-11e9-9df5-e057b6aa5f56.png)

Further, I propose the following changes to the PXL font (see image below). I can see from the comments in the pull request for the addition of the Italian language support that the creator of the font supplied the updated font files.

![image](https://user-images.githubusercontent.com/11942066/66311170-26b94200-e90e-11e9-8c0d-64d010c82598.png)

johanberntsson commented:
Hej Thomas

Tack för filerna och implementationen av danska. Vi har lagt in dessa i Ozmoo.

Vänliga hälsningar,
Johan Berntsson

On Mon, 7 Oct 2019 at 20:52, Thomas Bøvith <notifications@github.com> wrote:
>
> I have added support for Danish language to streams.asm and added a font file and a font license file. I changed the Swedish font file so that ä/Ä became æ/Æ and ö/Ö became ø/Ø (see the photo):
>
> ________________________________
>
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/johanberntsson/ozmoo/pull/5
>
> Commit Summary
>
> Add support for Danish language
> Add support for Danish language
>
> File Changes
>
> M asm/streams.asm (45)
> A fonts/system-da.fnt (0)
> A fonts/system-da.license.txt (10)
>
> Patch Links:
>
> https://github.com/johanberntsson/ozmoo/pull/5.patch
> https://github.com/johanberntsson/ozmoo/pull/5.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

thomasboevith commented:
Hej Johan,

Selv tak... og tak for ozmoo-pakken! Den kan nok bruges til mange sjove og
interessante projekter.

Med venlige hilsener,
Thomas


On Thu, Oct 10, 2019 at 12:41 AM Johan Berntsson <notifications@github.com>
wrote:

> Hej Thomas
>
> Tack för filerna och implementationen av danska. Vi har lagt in dessa i
> Ozmoo.
>
> Vänliga hälsningar,
> Johan Berntsson
>
> On Mon, 7 Oct 2019 at 20:52, Thomas Bøvith <notifications@github.com>
> wrote:
> >
> > I have added support for Danish language to streams.asm and added a font
> file and a font license file. I changed the Swedish font file so that ä/Ä
> became æ/Æ and ö/Ö became ø/Ø (see the photo):
> >
> > ________________________________
> >
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/johanberntsson/ozmoo/pull/5
> >
> > Commit Summary
> >
> > Add support for Danish language
> > Add support for Danish language
> >
> > File Changes
> >
> > M asm/streams.asm (45)
> > A fonts/system-da.fnt (0)
> > A fonts/system-da.license.txt (10)
> >
> > Patch Links:
> >
> > https://github.com/johanberntsson/ozmoo/pull/5.patch
> > https://github.com/johanberntsson/ozmoo/pull/5.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub, or mute the thread.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/johanberntsson/ozmoo/pull/5?email_source=notifications&email_token=AC3DRMRGJPNWZGAX2QEN4CLQNZM2TA5CNFSM4I6DN4RKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEAZVTBI#issuecomment-540236165>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC3DRMQCXS3CC2OYE6XXMV3QNZM2TANCNFSM4I6DN4RA>
> .
>
